Tasks/ArchiveFilesV2/_buildConfigs/Node20/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ArchiveFiles",
+  "version": "1.0.7",
+  "description": "Archive Files Task",
+  "main": "archivefilestask.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "@types/node": "^20.3.1",
+    "@types/mocha": "^9.1.1",
+    "azure-pipelines-task-lib": "^5.0.0-preview.0"
```

Comment:
todo: azure-pipelines-task-lib is compiled with node16 - we'll have a node20 version (not for this PR)